index.js

```diff
@@ -0,0 +1,45 @@
+function validatePassword(password) {
+  let lowercase = 'abcdefghijklmnopqrstuvwxyz'
+  let uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+  let special = '!@#$%^&*()-_+='
+  let numbers = '1234567890'
+  
+  let validLower = false
+  let validUpper = false
+  let validNumber = false
+  let validSpecChar = false
+  let validLength = false
+
+  for (let i = 0; i < password.length; i++) {
+    let currentLetter = password[i]
+
+    if (password.length >= 8) {
+      validLength = true
```

Comment:
can be moved out of the for loop. as password length does not change for each character of the password